src/Quiz.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import './App.css';
+import Demon_sprite from './data/spr/Demon_sprite.png'
+import Imp_sprite from './data/spr/Imp_sprite.png'
+import useSound from 'use-sound';
+import dssgtatk from './data/snd/dssgtatk.wav';
+// /static/media/dssgtatk.cd1a3ad1.wav
+
+function Quiz(props) {
+  // const props = {
+  //   image: "pathToPic",
+  //   sounds: [0, 1, 2]
+  // }
+
+  const [selectedButton, setSelectedButton] = useState(0);
+  const [isAnswered, setIsAnswered] = useState(false);
+  const [isDisabledButtons, setIsDisabledButtons] = useState(false);
+  const [hurlAcceleration, setHurlAcceleration] = useState(0);
+
+  const soundUrl1 = props.sounds[0];
+  const correctAnswer = 2;
+
+  const [play1] = useSound(
+    soundUrl1,
+    { volume: 0.25, interrupt: true, onend: () => setIsDisabledButtons(false) }
+  );
+
+  const soundUrl2 = props.sounds[1];
+
+  const [play2] = useSound(
+    soundUrl2,
+    {
+      volume: 0.25,
+      playbackRate: (soundUrl2.includes('dssgtatk') ? 0.8 + hurlAcceleration : 1),
```

Comment:
You may want to leave a comment before this code to describe what actually happens

Reply:
Done.